_helpers/task_executer_service.py

```diff
@@ -0,0 +1,126 @@
+import json
+import threading
+from concurrent.futures import as_completed
+from os import getenv
+
+from config import Config
+from config.mappings.assessment_mapping_fund_round import fund_round_data_key_mappings
+from db.queries import bulk_insert_application_record
+from fsd_utils.services.aws_extended_client import SQSExtendedClient
+
+
+class TaskExecutorService:
```

Comment:
Does all of this need redefining, as a lot of it is common with notify? The receive and delete portions should do the same thing, but we could provide a callback to do the processing?

Reply:
Fixed